pkg/plugins/resources/k8s/mapper_test.go

```diff
@@ -34,19 +34,6 @@ var _ = Describe("KubernetesStore template", func() {
 			Expect(res.GetNamespace()).To(Equal("core-ns"))
 		})
 
-		It("works with namespace in name on inference", func() {
```

Comment:
@lahabana is it a valid test case? Maybe I'm missing something, but InferenceMapper runs only on non-k8s store types, so how can it get a name with ns-suffix as an input?

Reply:
Hmm I'm not sure. Global k8s syncing a k8s zone maybe?